validator/bundle.py

```diff
@@ -36,7 +40,7 @@ def get_sub_type(self, discriminator: str) -> Optional["GraphqlType"]:
 @dataclass
 class Bundle:
 
-    graphql: list[dict[str, Any]]
+    graphql: Union[list[dict[str, Any]], dict[str, Any]]
```

Comment:
union types are always messy. how about you put the logic of dealing with both formats into `load_bundle` and only have one of those types here?

Reply:
I tried to keep the logic consistent across with qontract-server as well. Similar to qontract-server we can undo the changes here once the transition is done.